app/views/spotlight/talks/index.turbo_stream.erb

```diff
@@ -0,0 +1,20 @@
+
+<%= turbo_stream.update "talks_search_results", method: "morph" do %>
+  <div class="flex items-center flex-wrap gap-2 py-4">
+    <h2>Talks</h2>
+    <% if @talks.size.positive? %>
+      <span class="text-sm text-gray-500">(<%= @talks.size %> of <%= @talks_count %>)</span>
+      <%= link_to talks_path(s: search_query), class: "link text-sm text-gray-500", target: "_top" do %>
```

Comment:
What do you think about displaying the search query on the talks list after the user clicks See Results? For example, I searched for 'postgres,' and I can see the highlighted text, but maybe showing the query explicitly would give more context to the user. Do you think that's unnecessary?

![image](https://github.com/user-attachments/assets/055efc3f-2207-4ede-80aa-814a23bdc872)

The same for speakers and events. Here I searched for "Amanda":
![image](https://github.com/user-attachments/assets/386c362d-6df7-4551-bfcf-2085552e6bbd)

Reply:
yes very good idea, this is a rather common UX and fairly easy to implement

Reply:
![CleanShot 2024-11-17 at 22 17 49@2x](https://github.com/user-attachments/assets/445ffd54-30f9-4736-bca9-0ce30056f39a)

Reply:
I have updated the title like that ☝️